rules/integrations/aws/privilege_escalation_aws_iam_administratoraccess_policy_attached_to_group.toml

```diff
@@ -0,0 +1,143 @@
+- If you suspect the account has been compromised, scope potentially compromised assets by tracking servers, services, and data accessed by the account in the last 24 hours.
+    - Determine what other API calls were made by the user.
+    - Assess whether this behavior is prevalent in the environment by looking for similar occurrences involving other users.
+
+### False positive analysis
+
+- False positives may occur due to the intended usage of the IAM `AdministratorAccess` managed policy. Verify the `aws.cloudtrail.user_identity.arn` should have the `iam:AttachUserPolicy` permission and that the `target.userName` should be given full administrative access.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Disable or limit the account during the investigation and response.
+    - Rotate user credentials
+    - Remove the `AdministratorAccess` policy from the affected group(s)
+- Identify the possible impact of the incident and prioritize accordingly; the following actions can help you gain context:
+    - Identify the account role in the cloud environment.
+    - Assess the criticality of affected services and servers.
+    - Work with your IT team to identify and minimize the impact on users.
+    - Identify if the attacker is moving laterally and compromising other accounts, servers, or services.
+    - Identify any regulatory or legal ramifications related to this activity.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are identified. 
+    - Rotate secrets or delete API keys as needed to revoke the attacker's access to the environment. 
+    - Work with your IT teams to minimize the impact on business operations during these actions.
+- Check if unauthorized new users were created, remove unauthorized new accounts, and request password resets for other IAM users.
+- Consider enabling multi-factor authentication for users.
+- Review the permissions assigned to the implicated user to ensure that the least privilege principle is being followed.
+- Implement security best practices [outlined](https://aws.amazon.com/premiumsupport/knowledge-center/security-best-practices/) by AWS.
+- Take the actions needed to return affected systems, data, or services to their normal operational levels.
+- Identify the initial vector abused by the attacker and take action to prevent reinfection via the same vector.
+- Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the mean time to respond (MTTR).
+"""
+references = [
+    "https://docs.aws.amazon.com/IAM/latest/APIReference/API_AttachGroupPolicy.html",
+    "https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AdministratorAccess.html",
+    "https://hackingthe.cloud/aws/exploitation/iam_privilege_escalation/",
+]
+risk_score = 73
+rule_id = "df919b5e-a0f6-4fd8-8598-e3ce79299e3b"
+severity = "high"
```

Comment:
```suggestion
risk_score = 47
rule_id = "df919b5e-a0f6-4fd8-8598-e3ce79299e3b"
severity = "medium"
```

Reply:
As this might as well be an administrative task, I would personally reduce risk to medium/low.